docs/copy-product.md

```diff
@@ -0,0 +1,66 @@
+## Copy Product
+
+This command makes a copy of the referenced product.
+Target product will be searched by source product system name.
+System name can be overriden with `--target_system_name` option.
+If a product with the selected `system_name` is not found, it will be created.
+
+Components of the backend being copied:
```

Comment:
Can we document/explain the copy behavior when the destination product exists? (merge behavior)

Reply:
:+1: